### PR TITLE
[WIP] Release 0.2.1: Changelog

### PR DESCRIPTION
**Pull request by ax3l**
Prepares the changelog for the 0.2.1 release.

- **README: Document PIC extensions**: Highlight extensions we already added that are not in the
standard PIC cycle.
- **REFERENCE: Add Format for Markdown**: Avoids scrumbling the text when opening the file directly
on a markdown viewer (online).
- **Release 0.2.1: Changelog**
  - #1659

@ComputationalRadiationPhysics/picongpu-maintainers please have a look and comment inline. If everything is ok, please approve.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,26 @@
 Change Log / Release Log for PIConGPU
 ================================================================
 
+0.2.1
+-----
+**Date:** 2016-11-28
+
+Fix potential Deadlock in Checkpoints & QED-RR
```
Comment (PrometheusPi) [inline CHANGELOG.md]: Fix potential deadlock in checkpoints & highlight QED nonlinear Thomson scattering

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,26 @@
...
+
+This releases fixes a potential deadlock encountered during checkpoints and
+initialization. Furthermore, we forgot to highlight that the 0.2.0 release
+also included a QED radiation reaction scheme (based on the work of
```
Comment (PrometheusPi) [inline CHANGELOG.md]: not only radiation reaction scheme but nonlinear Thomson scattering module

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,26 @@
...
+A. Gonoskov et al., PRE 92, 2015).
+
+### Changes to "0.2.0"
+
+**Bug Fixes:**
+ - potential event system deadlock init/checkpoints #1659
+
+Thank you to René Widera for spotting & fixing and Heiko Burau for the QED-RR
```
Comment (PrometheusPi) [inline CHANGELOG.md]: RR->scattering

File: README.md [hunk 4]
```diff
@@ -26,12 +26,18 @@ Its features include:
 - macro-particle form factors ranging from NGP (0th order), CIC (1st),
   TSC (2nd), PSQ (3rd) to P4S (4th)
 
-Besides the electro-magnetic PIC algorithm, we developed a wide range of tools
-and diagnostics, e.g.:
+and the electro-magnetic PIC algorithm is further self-consistently coupled to:
+- classical radiation reaction (DOI: 10.1016/j.cpc.2016.04.002)
+- QED radiation reaction (DOI: 10.1103/PhysRevE.92.023305)
```
Comment (PrometheusPi) [inline README.md]: nonlinear Thomson- and Compton scattering

File: CHANGELOG.md (hunk 1 above)
Comment (ax3l) [inline CHANGELOG.md]: That's gone be the title, around 40-60 chars would be ideal...

File: README.md (hunk 4 above)
Comment (ax3l) [inline README.md]: - QED synchrotron radiation (photon emission)

File: CHANGELOG.md (hunk 2 above)
Comment (ax3l) [inline CHANGELOG.md]: also included a QED synchrotron emission scheme

File: CHANGELOG.md (hunk 3 above)
Comment (ax3l) [inline CHANGELOG.md]: QED synchrotron photon emission implementation

**Pull request merged**